app/views/layouts/header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class='page-header'>
+  
+  <div class='logo'> 
+    <a href="/">GIVING PROFILES</a>
+  </div>
+
+  <!-- This is just a place holder for now. -->
+  <div class='header-search'>
+    <%= fa_icon "search", text: "Search..." %>
+  </div>
+
+</header>
```

Comment:
If we are installing bootstrap, shouldn't we leverage some of it by using some of its components, such as https://getbootstrap.com/docs/4.3/components/navbar/

Reply:
I thought about that, but I just decided to keep it simple for now since the search is just a place holder for now. Also, I didn't want any of the extra styling it provided. But I can look more into it.

Reply:
I'm just going to leave it for now. I can play around with it some more later on a different pr.

Reply:
I think we should leverage the framework if we took the time to research and decide which one to use. The story is to implement the styling for the header, which in my opinion has room to explore. 

In a sense, if bootstrap is to be the standard for our style I think we should rather follow it on the ticket focused on implementing the design.

Reply:
I'm not working on this anymore at the moment because I realized that Emily does some things with header on her branch. I'm just going to wait for her branch to be merged in. I'll just pic up a different task if she completes the header in her branch.

Reply:
Ah - her portion of the work which I paired with briefly doesn't actually touch (or shouldn't) any of the header stuff you worked on. Code wise there are some concerns on switching the css structure, but other than that it shouldn't be a blocker for you

And that's fine - I just think that refactoring this to use bootstrap's CSS components doesn't really justify it to be another ticket

Reply:
@sanghun89, @markdamobi and I had concerns about merging conflicts when both of our PRS would go to master since we redid my file structure to be the _user_ header instead of just the home page. And he worked on header stuff as well. If you reference the slack convo (I'll tag you in it so you know where to look) we had discussed adding some of his code to my branch in order to avoid that. Do you think this is still necessary? Can you offer your thoughts there since you have an insight into how we structured that?

Reply:
yeah we should prob use the bootstrap component but I'm not going to hold this up to get that in, couple things waiting on this.